mirror-main.go

```diff
@@ -174,8 +180,8 @@ func doMirror(sURLs mirrorURLs, progressReader *progressBar, accountingReader *a
 	return sURLs
 }
 
-// doMirrorFake - Perform a fake mirror to update the progress bar appropriately.
-func doMirrorFake(sURLs mirrorURLs, progressReader *progressBar) mirrorURLs {
+// doMirrorFakeProgressUpdate - Update the progress bar to represent a fake mirror
+func doMirrorFakeProgressUpdate(sURLs mirrorURLs, progressReader *progressBar) mirrorURLs {
```

Comment:
During 'fake' it means that we are not updating the progress bar, we should update rather than returning dummy.